examples/compiler/03_iet.ipynb

```diff
@@ -0,0 +1,547 @@
+      "} ;\n",
+      "\n",
+      "\n",
+      "int Kernel(float *restrict u_new_vec, const int time_M, const int time_m, struct profiler* timers, const int x_M, const int x_m, const int x_size, const int y_M, const int y_m, const int y_size)\n",
+      "{\n",
+      "  float (*restrict u_new)[x_size + 1 + 3][y_size + 1 + 3] __attribute__((aligned(64))) = (float (*)[x_size + 1 + 3][y_size + 1 + 3]) u_new_vec;\n",
+      "  /* Flush denormal numbers to zero in hardware */\n",
+      "  _MM_SET_DENORMALS_ZERO_MODE(_MM_DENORMALS_ZERO_ON);\n",
+      "  _MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_ON);\n",
+      "  for (int time = time_m, t0 = (time)%(2), t1 = (time + 1)%(2); time <= time_M; time += 1, t0 = (time)%(2), t1 = (time + 1)%(2))\n",
+      "  {\n",
+      "    struct timeval start_section0, end_section0;\n",
+      "    gettimeofday(&start_section0, NULL);\n",
+      "    for (int x = x_m; x <= x_M; x += 1)\n",
+      "    {\n",
+      "      #pragma omp simd\n",
+      "      for (int y = y_m; y <= y_M; y += 1)\n",
+      "      {\n",
+      "        u_new[t1][x + 3][y + 3] = u_new[t0][x + 3][y + 3] + 2;\n",
+      "      }\n",
+      "    }\n",
+      "    gettimeofday(&end_section0, NULL);\n",
+      "    timers->section0 += (double)(end_section0.tv_sec-start_section0.tv_sec)+(double)(end_section0.tv_usec-start_section0.tv_usec)/1000000;\n",
+      "  }\n",
+      "  return 0;\n",
+      "}\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "equation = Eq(u_new.forward, u_new + 2)\n",
+    "op = Operator(equation)\n",
+    "print(op)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, let's run the operator."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Operator `Kernel` run in 0.00 s\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[[[0. 0. 0. 0. 0. 0. 0.]\n",
+      "  [0. 0. 0. 0. 0. 0. 0.]\n",
+      "  [0. 0. 0. 0. 0. 0. 0.]\n",
+      "  [0. 0. 0. 5. 5. 5. 0.]\n",
+      "  [0. 0. 0. 5. 5. 5. 0.]\n",
+      "  [0. 0. 0. 5. 5. 5. 0.]\n",
+      "  [0. 0. 0. 0. 0. 0. 0.]]\n",
+      "\n",
+      " [[0. 0. 0. 0. 0. 0. 0.]\n",
+      "  [0. 0. 0. 0. 0. 0. 0.]\n",
+      "  [0. 0. 0. 0. 0. 0. 0.]\n",
+      "  [0. 0. 0. 7. 7. 7. 0.]\n",
+      "  [0. 0. 0. 7. 7. 7. 0.]\n",
+      "  [0. 0. 0. 7. 7. 7. 0.]\n",
+      "  [0. 0. 0. 0. 0. 0. 0.]]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "op.apply(time_M=2)\n",
+    "print(u_new.data_with_halo)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There is a region that surrounding the halo region, that region is named `padding`. Padding can be used for data alignment. By default there is no padding, but it can be changed by passing a value in `padding`, let's see below:"
```

Comment:
Ι think it would be better to say:
     "There is a region surrounding the halo region, named `padding`. Padding can be used for data alignment. By default, there is no padding, but it can be changed by passing a value in `padding`, as shown below:"